vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/ChartVariant.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.charts;
+
+/**
+ * Set of theme variants applicable for {@code vaadin-chart} component.
+ */
+public enum ChartVariant {
+    LUMO_GRADIENT("gradient"), LUMO_MONOTONE("monotone"), LUMO_CLASSIC(
+            "classic");
```

Comment:
Let's also add `MATERIAL` variants because Material theme also has corresponding [styles](https://github.com/vaadin/web-components/blob/aa79465da8f3d9be92e22bdd04ef2a5fdb1d22a1/packages/charts/theme/material/vaadin-chart-styles.js#L22-L63) in the web component.

Reply:
Added the `MATERIAL` variants. Now both `LUMO` and `MATERIAL` theme variants have the same values